### PR TITLE
Added the date a downloaded eFolder will be deleted.

### PR DESCRIPTION
**Pull request by mdbenjam**
Removed the text saying that downloads are deleted in 72 hours and added specific delete dates to each download.

Fixes https://github.com/department-of-veterans-affairs/appeals-pm/issues/781

**Test Plan**
-Run the test suite
-Check the text on the page:
screen shot 2016-10-25 at 11 37 45 am

### CODE REVIEW

File: app/models/download.rb [hunk 1]
```diff
@@ -133,6 +133,11 @@ def user_id_string
     "#{user_id} (Station #{user_station_id})"
   end
 
+  def expriation_day
+    (completed_at + HOURS_UNTIL_EXPIRY.hours).strftime("%m/%d")
```
Comment (joofsh) [inline app/models/download.rb]: nil check please

Comment (joofsh) [inline app/models/download.rb]: created_at

**Comment by mdbenjam** [comment]
We can't make the color light gray since it violates the color contrast requirements in 508 rules.

File: app/views/downloads/new.html.erb [hunk 2]
```diff
@@ -54,8 +54,7 @@
   </div>
 
   <% if recent_downloads.exists? %>
-    <h2 class="ee-recent-searches">History</h2>
-    <p class="ee-recent-searches-note">Note: eFolders remain in your history for <b>72 hours</b></p>
+    <h2 class="ee-recent-searches">Download History</h2>
```
Comment (joofsh) [inline app/views/downloads/new.html.erb]: I think we want it to stay "History"

File: app/views/downloads/new.html.erb [hunk 3]
```diff
@@ -68,7 +67,12 @@
         <tr id="download-<%= download.id %>">
           <td>
             <%= download.veteran_name %>
-            <span class="cf-subtext">(<%= download.file_number %>)</span>
+            <span class="cf-subtext">
+              (<%= download.file_number %>)
+              <% if download.expriation_day %>
```
Comment (joofsh) [inline app/views/downloads/new.html.erb]: typo I think `expriation` => `expiration`

**Comment by joofsh** [comment]
@mdbenjam Bump

**Comment by mdbenjam** [comment]
@joofsh Whoops, thanks for the bump. Just fixed the typo.

**Pull request merged**